Formula/zpaq.rb

```diff
@@ -15,20 +17,52 @@ class Zpaq < Formula
     sha256 "a2d2215b51cc18370f5ebe700160d1d7143ce5d680b2365a5adb4bb1622d06fc" => :mountain_lion
   end
 
+  devel do
+    url "http://mattmahoney.net/dc/zpaq713.zip"
+    sha256 "9120cf4fb1afdecea3ac4f690d7b0577f7cb004ca6b152856edd8ac444f0d919"
+    version "7.13"
+  end
+
+  resource("test") do
```

Comment:
Style nit but we don't normally paren-wrap resources at this point. I'd prefer to keep that consistent unless you feel like changing them all :trollface:.